packages/logger/lib/logger.js

```diff
@@ -171,6 +200,15 @@ class Logger {
 				messageKey: 'message', // This is for backwards compatibility with our existing logs
 				timestamp: withTimestamps
 			};
+			if (PRETTIFICATION_AVAILABLE) {
+				pinoOptions.transport = {
+					target: 'pino-pretty',
+					options: {
+						colorize: true,
+						messageKey: 'message'
+					}
+				};
+			}
```

Comment:
Conditional property assignment means you could include the `transport` property in the initial assignment of `pinoOptions`, but looking at it, what you've written is much easier to follow, so ignore this comment 😄 

```js
const pinoOptions = {
	base: {},
	formatters: {
		level(label) {
			return { level: label };
		}
	},
	messageKey: 'message', // This is for backwards compatibility with our existing logs
	timestamp: withTimestamps,
	...(PRETTIFICATION_AVAILABLE && {
		transport: {
			target: 'pino-pretty',
			options: {
				colorize: true,
				messageKey: 'message'
			}
		}
	})
};
```

Reply:
Yeah call me old fashioned but I love a regular old `if` 😁